StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSession.swift

```diff
@@ -69,6 +69,28 @@ public extension StripeAPI {
             }
         }
 
+        enum Status: String, SafeEnumCodable, Equatable {
+            case pending
+            case succeeded
+            case failed
+            case cancelled
+            case unparsable
+        }
```

Comment:
backend model to double check: 
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/api/resource/enums.rb#L11-14